src/ExecBlock/ExecBlock.h

```diff
@@ -164,6 +164,10 @@ class ExecBlock {
      */
     uint16_t splitSequence(uint16_t instID);
 
+    rword getDataBlockBase() const {
+        return (rword) dataBlock.base();
```

Comment:
use ``static_cast``

Reply:
cannot use static_cast on void* to a rword. replaced by a reinterpreted_cast